lib/Rest/HttpResponse.cpp

```diff
@@ -349,9 +349,26 @@ void HttpResponse::addPayloadInternal(VPackSlice output, size_t inputLength,
     }
     default: {
       setContentType(rest::ContentType::JSON);
+      
+      /// dump options contain have the escapeUnicode attribute set to true
+      /// this allows dumping of string values as plain 7-bit ASCII values.
+      /// for example, the string "möter" will be dumped as "m\u00F6ter".
+      /// this allows faster JSON parsing in some client implementations,
+      /// which speculate on ASCII strings first and only fall back to slower
+      /// multibyte strings on first actual occurrence of a multibyte character.
+      
+      VPackOptions tmpOpts = *options;
+      tmpOpts.escapeUnicode = true;
+      
       if (_generateBody) {
-        arangodb::basics::VelocyPackDumper dumper(_body.get(), options);
-        dumper.dumpValue(output);
+        
+        // convert object to string
+        VPackStringBufferAdapter buffer(_body->stringBuffer());
+
+        // usual dumping -  but not to the response body
```

Comment:
I think the comment is misleading. We _are_ dumping the response into `_body`, which is the StringBuffer for the response body 
```suggestion
        // usual dumping into the response body
```